src/DetailsView/components/tab-stops-failed-instance-section.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { css } from '@uifabric/utilities';
+import { CollapsibleComponentCardsDeps } from 'common/components/cards/collapsible-component-cards';
+import { ResultSectionTitle } from 'common/components/cards/result-section-title';
+import { VisualizationScanResultData } from 'common/types/store-data/visualization-scan-result-data';
+import { AdhocTabStopsTestViewDeps } from 'DetailsView/components/adhoc-tab-stops-test-view';
+import { requirements } from 'DetailsView/components/tab-stops/requirements';
+import { TabStopsFailedCounter } from 'DetailsView/tab-stops-failed-counter';
+import { TabStopsRequirementResult } from 'DetailsView/tab-stops-requirement-result';
+import {
+    TabStopsRequirementsWithInstances,
+    TabStopsRequirementsWithInstancesDeps,
+} from 'DetailsView/tab-stops-requirements-with-instances';
+import * as React from 'react';
+import * as styles from './tab-stops-failed-instance-section.scss';
+
+export type TabStopsFailedInstanceSectionDeps = AdhocTabStopsTestViewDeps &
```

Comment:
Not sure if this was intentional or a mistyped copy/paste but we should only have deps for components that are children of the current component (TabStopsFailedInstanceSection in this case).